plugins/headlamp-plugin/template/.vscode/settings.json

```diff
@@ -0,0 +1,26 @@
+// Place your settings in this file to overwrite default and user settings.
+{
+  "files.exclude": {
+    "build": false // set this to true to hide the "dist" folder with the compiled JS files
```

Comment:
So this hides this folder from the file tree in VSCode? I am not sure I'd like this change for me as a developer. As long as it doesn't get in my way in git (i.e. it's in .gitignore), I'm fine and find it useful to have things available in the file tree. Same logic for node_modules for example.

Reply:
removed this